cpp/models/ide_secir/parameters_io.cpp

```diff
@@ -0,0 +1,231 @@
+/*
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Lena Ploetzke, Anna Wendler
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "ide_secir/parameters_io.h"
+#include "memilio/config.h"
+
+#ifdef MEMILIO_HAS_JSONCPP
+
+#include "ide_secir/model.h"
+#include "ide_secir/infection_state.h"
+#include "memilio/math/eigen.h"
+#include "memilio/io/epi_data.h"
+#include "memilio/io/io.h"
+#include "memilio/utils/date.h"
+
+#include <string>
+#include <cmath>
+
+namespace mio
+{
+namespace isecir
+{
+
+// We assume that we start the simulation at time 0 and want to compute the necessary flows
```

Comment:
I think it would be fairly easy to adapt this, but it is not a problem at all to leave this assumption.